src/main/java/com/faforever/client/game/TeamCardController.java

```diff
@@ -59,25 +59,31 @@ public static void createAndAdd(ObservableMap<? extends String, ? extends List<S
 
       TeamCardController teamCardController = uiService.loadFxml("theme/team_card.fxml");
       teamCardController.setPlayersInTeam(entry.getKey(), players,
-          player -> new Rating(player.getGlobalRatingMean(), player.getGlobalRatingDeviation()));
+          player -> new Rating(player.getGlobalRatingMean(), player.getGlobalRatingDeviation()), true);
       teamsPane.getChildren().add(teamCardController.getRoot());
     }
   }
 
-  public void setPlayersInTeam(String team, List<Player> playerList, Function<Player, Rating> ratingProvider) {
+  public void setPlayersInTeam(String team, List<Player> playerList, Function<Player, Rating> ratingProvider, boolean useRoundedRating) {
     int totalRating = 0;
     for (Player player : playerList) {
       // If the server wasn't bugged, this would never be the case.
       if (player == null) {
         continue;
       }
       PlayerCardTooltipController playerCardTooltipController = uiService.loadFxml("theme/player_card_tooltip.fxml");
-      playerCardTooltipController.setPlayer(player);
+      int playerRating = RatingUtil.getRating(ratingProvider.apply(player));
+      if (useRoundedRating) {
+        playerCardTooltipController.setPlayer(player);
+      } else {
+        playerCardTooltipController.setPlayerWithRating(player, playerRating);
+      }
```

Comment:
This `if` assumes that `setPlayer` will round the rating, but that might not be true. Neither is the assumption that `setPlayerWithRating` will not round it.

By passing the new enum parameter, we fix this.